### PR TITLE
Various translation related additions

### PR DESCRIPTION
**Pull request by VortexAcherontic**
Hello there, by this Pull Request I'd like to merge an updated boxbuddyrs.pot file including text I added in the previous PR.

In addition I added a German translation (native) and updated uk_UA, ru_RU and it_IT using DeepL to include the newly introduced messages as well as adding some which where already missing.

Furthermore I issued a small update on the create-pot.sh script to work on different users by using `$HOME` instead of a fixed user path.
Also I added a quick dependency check which tests if `xtr` is installed in `$HOME/.cargo/bin/` and if not will run `cargo install xtr` and then generate the pot file.

Kind regards,  
V

### CODE REVIEW

**Comment by Dvlv** [comment]
Thank you!

I don't think I meant to commit the `create-pot` script, but it's nice now to have it actually usable.
Regenerating the .pot is going to remove my context comments, so I guess I need to move them to a separate file.

Not sure how I feel about machine-translations, but I suppose it's fine just for these couple of strings.

I'll merge this when I've got the Custom Home Dir stuff all figured out.

**Comment by VortexAcherontic** [comment]
> Regenerating the .pot is going to remove my context comments, so I guess I need to move them to a separate file.

I apologize for this did not payed proper attention to this. I re-added the comments.

> Not sure how I feel about machine-translations, but I suppose it's fine just for these couple of strings.

I agree at some point the question arises if it is better to have no translation or possibly broken translations 😄   
Maybe @albanobattistella could revisit the changes on it_IT and @vovkiv could revisit the changes on uk_UA and ru_RU?

**Comment by Vovkiv** [comment]
Oh, wowie
Yes, I will maintain Ukrainian and Russian translation myself, no need to touch it by machine translation in future, just ping by @Vovkiv.

File: po/ru_RU/LC_MESSAGES/ru_RU.po [hunk 1]
```diff
@@ -7,125 +7,142 @@ msgid ""
...
 
-#: src/main.rs:87
+#: src/main.rs:83
 msgid "Create A Distrobox"
 msgstr "Создать Distrobox"
 
-#: src/main.rs:110
+#: src/main.rs:89
+msgid "About BoxBuddy"
+msgstr "О компании BoxBuddy"
```
Comment (Vovkiv) [inline po/ru_RU/LC_MESSAGES/ru_RU.po:28]: This should be "Про BoxBuddy"

File: po/ru_RU/LC_MESSAGES/ru_RU.po [hunk 2]
```diff
@@ -7,125 +7,142 @@ msgid ""
...
 
-#: src/main.rs:295
+#: src/main.rs:291
 msgid "Name"
 msgstr "Название"
 
-#: src/main.rs:307
+#: src/main.rs:303
+msgid "Home Directory (Leave blank for default)"
+msgstr "Домашняя директория (Оставьте пустым по умолчанию)"
```
Comment (Vovkiv) [inline po/ru_RU/LC_MESSAGES/ru_RU.po:72]: This should be "Домашняя папка (оставьте пустой для настроек по-умолчанию)"

File: po/uk_UA/LC_MESSAGES/uk_UA.po [hunk 3]
```diff
@@ -7,125 +7,142 @@ msgid ""
...
 
-#: src/main.rs:295
+#: src/main.rs:291
 msgid "Name"
 msgstr "Назва"
 
-#: src/main.rs:307
+#: src/main.rs:303
+msgid "Home Directory (Leave blank for default)"
+msgstr "Домашній каталог (залишити порожнім за замовчуванням)"
```
Comment (Vovkiv) [inline po/uk_UA/LC_MESSAGES/uk_UA.po:72]: This should be "Домашня тека (залиште порожньою для налаштувань за замовченням)"

**Review by Vovkiv — CHANGES_REQUESTED** [review]
This strings were translated very poorly, but overall seems fine.

**Comment by Vovkiv** [comment]
@Dvlv 
> I don't think I meant to commit the `create-pot` script, but it's nice now to have it actually usable. Regenerating the .pot is going to remove my context comments, so I guess I need to move them to a separate file.

I think you meant to add this comments in source code, near line where string taken, with `TRANSLATORS: ` keyword, refer to rust gettext implementation or GNU gettext https://www.gnu.org/software/gettext/manual/gettext.html#PO-Files

**Comment by VortexAcherontic** [comment]
Many thanks @Vovkiv I included the suggested changes.

**Pull request merged**